### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by trwalke**
Update changelog for 4.49.0

Fixes #

**Changes proposed in this request**

**Testing**

**Performance impact**

**Documentation**
- [ ] All relevant documentation is updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+4.49.0
+==========
+### New Features
+- MSAL will now use `<region>.login.microsoft.com` when using regional ESTS-R. See [3252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3252)
+- Added Support for Account Discovery when using Wam broker. See [3458](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3458)
+- Added support for app service system assigned managed identity (Managed Service Identity). See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754)
+- Added IMDS managed identity source to Managed Service Identity integration in MSAL. See [3829](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3829)
```
Comment (pmaytak) [inline CHANGELOG.md:7]:
```suggestion
- Added ability to acquire tokens using system and user-assigned managed identities in App Services and IMDS-enabled environments. See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754) and [3829](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3829)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+4.49.0
+==========
+### New Features
+- MSAL will now use `<region>.login.microsoft.com` when using regional ESTS-R. See [3252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3252)
+- Added Support for Account Discovery when using Wam broker. See [3458](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3458)
```
Comment (pmaytak) [inline CHANGELOG.md:5]:
```suggestion
- Added support for acquiring Work and School accounts when calling `GetAccounts` using the new Broker preview. See [3458](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3458)
```

**Review by pmaytak — APPROVED** [review]

**Review by neha-bhargava — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+- Added support for app service system assigned managed identity (Managed Service Identity). See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754)
```
Comment (gladjohn) [inline CHANGELOG.md:6]: [Managed identities for Azure resources](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) provide Azure services with an automatically managed identity in Azure Active Directory. You can use this identity to authenticate to any service that supports Azure AD authentication, without having credentials in your code. MSAL .NET now supports acquiring token for Managed Identities for Azure App Service and Azure Virtual Machines. See [3754](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3754) and [3829](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3829).  Use `WithManagedIdentity` method on the `AcquireTokenForClient` API to get MSI token in MSAL. This is an experimental feature and may change in the future versions of MSAL

**Review by gladjohn — APPROVED** [review]

**Comment by gladjohn** [comment]
@trwalke we may also need to add the GC fix we took for the new broker.

**Comment by trwalke** [comment]
> @trwalke we may also need to add the GC fix we took for the new broker.
@gladjohn 
Added it. Please verify that it is correct.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+
+### Supportability
+- Enabling MSAL Runtime logging integration. See [3575](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3575)
+
+### Bug Fixes
+- Optimizing MSAL cache key logic to improve performance. See [3393](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3393)
+- Update Broker to use Interop 0.13.3 to resolve crash resulting from garbage collection when using WAM broker. See [3868](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3868)
```
Comment (gladjohn) [inline CHANGELOG.md:17]:
```suggestion
- Update new WAM preview broker to use Microsoft.Identity.Client.NativeInterop [version 0.13.3](https://www.nuget.org/packages/Microsoft.Identity.Client.NativeInterop/0.13.3) to resolve a crash resulting from a managed delegate being garbage collected. See [3868](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3868)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
+4.49.0
+==========
+### New Features
+- MSAL will now use `<region>.login.microsoft.com` when using regional ESTS-R. See [3252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3252)
+- Added support for acquiring Work and School accounts when calling `GetAccounts` using the new Broker preview. See [3458](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3458)
+- Added the ability to disable Instance Discovery/Authority validation using `WithInstanceDiscovery(bool enableInstanceDiscovery)`. See [3775](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3775)
+- Added new APIs to acquire authentication data from WWW-Authenticate and Authentication-Info request headers. This will provide additional support for Proof-of-Possession. See [3026](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3026)
+
+### Expiremental Features
```
Comment (gladjohn) [inline CHANGELOG.md:9]:
```suggestion
### Experimental Features
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,21 @@
+4.49.0
+==========
+### New Features
+- MSAL will now use `<region>.login.microsoft.com` when using regional ESTS-R. See [3252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3252)
```
Comment (gladjohn) [inline CHANGELOG.md:4]:
```suggestion
- MSAL will now use `<region>.login.microsoft.com` when using regional ESTS-R for public cloud. See [3252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3252)
```

**Review by gladjohn — APPROVED** [review]

**Comment by gladjohn** [comment]
> > @trwalke we may also need to add the GC fix we took for the new broker.
> > @gladjohn
> > Added it. Please verify that it is correct.

reviewed, thanks @trwalke.

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**